### PR TITLE
feat(trackers): remove all social trackers, use links to web share intent

### PR DESCRIPTION
**Pull request by clacote**

### CODE REVIEW

File: _includes/ninjasquad/social-buttons [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<a class="twitter-share-button"
+   title="Share on Twitter"
+   target="social"
+   href="https://twitter.com/intent/tweet?text={{ post.title | default: page.title | url_encode }}%20{{ post.description | default: page.description | url_encode }}&url={{ post.url | default: page.url | absolute_url | url_encode }}&related=NinjaSquad,agnes_crepet,cedric_exbrayat,clacote,jbnizet"
```
Comment (clacote) [inline _includes/ninjasquad/social-buttons]: 😅 (I'm now fluent in Jekyll + Liquid + Ruby)
use `page` or `post` variable (thanks to `default` Liquid filter) for this fragment to work in both index page iterating on `post` and "post" page.

**Comment by clacote** [comment]
Not sure if this truly works. It works for Twitter, but Facebook and Google+ are currently refusing the intent (Facebook displays an error, and G+ displays no link to be shared). I hope it is just because, in dev, the link to be shared is on `http://localhost:4000`, thus Google+ and Facebook cannot slurp it.

File: _includes/ninjasquad/social-buttons (hunk 1 above)
Comment (cexbrayat) [reply _includes/ninjasquad/social-buttons]: I admit this is impressive 👏

**Comment by cexbrayat** [comment]
LGTM 👌

**Comment by clacote** [comment]
As hoped, G+ and Facebook sharing work fine once deployed to production and served on a true domain.

**Pull request merged**